vta/runtime/runtime.cc

```diff
@@ -329,7 +442,7 @@ class BaseQueue {
   // End location of current SRAM write in FIFO mode
   uint32_t sram_end_{0};
   // The buffer in DRAM
-  std::vector<T> dram_buffer_;
+  std::vector<T, AlignmentAllocator<T, 64>> dram_buffer_;
```

Comment:
I'm thinking we may want to make 64 a preprocessor constant

Reply:
done